modules/registry-checking-the-status-of-registry-pods.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * registry/accessing-the-registry.adoc
+
+[id="checking-the-status-of-registry-pods_{context}"]
+= Checking the status of the registry Pods
+
+As a cluster administrator you can list the image registry Pods running in the `openshift-image-registry` project and check their status.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. List the Pods in the `openshift-image-registry` project and view their status:
+
```

Comment:
Looks like an extra line between the step and the + before the code block

Reply:
Good catch, thank you! Resolved.